src/views/edition/Edition.vue

```diff
@@ -55,23 +107,80 @@ export default Vue.extend({
         next();
     },
     methods: {
+        openPermissionModal() {
+            console.log('ffff');
+            this.$root.$emit('bv::show::modal', 'permissionModal');
+        },
+        versionString(ver: EditionInfo) {
+            return ver.name;
+        },
         getPage(url: string) {
             if (url.endsWith('artefacts')) {
                 this.page = 'artefacts';
             } else {
                 this.page = 'imaged-objects';
             }
-        }
+        },
     },
 });
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
+@import '@/assets/styles/_variables.scss';
+@import '@/assets/styles/_fonts.scss';
+.btns-permiss {
+    float: right;
+}
+.btns-permiss button{
+    background-color: #bdbdbd;
```

Comment:
Shouldn't a lot of the colors come from your _variables.scss in the styles folder?  It seems to that would be safer in the long run.